test/compilable/ddoc4899.d

```diff
@@ -4,8 +4,6 @@
 /*
 TEST_OUTPUT:
 ---
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray '('. This may cause incorrect Ddoc output. Use $(LPAREN) instead for unpaired left parentheses.
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
```

Comment:
The change here is that previously `foo:)` was interpreted as a tag. But now, given that there's no whitespace between the colon and the close paren, the text is not interpreted as a tag anymore, the closing paren becomes legit, and the warning text disappears altogether. What am I missing?